notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        options=[d for d in core.available_devices if "GPU" not in d] + ["AUTO"],
</pre><p>why do you exclude GPU for inference? Stable Diffusion does not have issues for GPU running</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
fixed!